server/svix-server/src/v1/endpoints/application.rs

```diff
@@ -193,20 +194,40 @@ const LIST_APPLICATIONS_DESCRIPTION: &str = "List all of the organization's appl
 
 async fn list_applications(
     State(AppState { ref db, .. }): State<AppState>,
-    pagination: ValidatedQuery<Pagination<ApplicationId>>,
+    ValidatedQuery(pagination): ValidatedQuery<Pagination<ReversibleIterator<ApplicationId>>>,
+    ValidatedQuery(Ordering { order }): ValidatedQuery<Ordering>,
```

Comment:
Why isn't `order` just another field on `pagination`? It seems like they'll always go together, no?

Reply:
1. While a niche use-case I think ordering can have uses for non-paginated results too.
2. I can remove it if you think the above is too unlikely to happen. However, since `Ordering` isn't being added in this PR, it's only a usage of it, is it ok if we get this merged and I can merge it with Pagination in a separate one?